frontend/public/env.js

```diff
@@ -0,0 +1,2 @@
+window.localStorage.setItem('fam_api_base_url', 'https://341ihp76l2.execute-api.ca-central-1.amazonaws.com/test')
```

Comment:
Just quick question.
I understand at the moment this is hardcoded.
Will later the value come from Terraform?

Reply:
Yes, it will come from the pipeline.